app/kumactl/data/install/k8s/metrics/grafana/grafana.yaml

```diff
@@ -130,6 +130,31 @@ data:
         editable: true
 ---
 apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: provisioning-dashboards
+  namespace: {{ .Namespace }}
+  labels:
+    app: grafana
+data:
+  dashboards.yaml: |
+    apiVersion: 1
+
+    providers:
+    - name: 'Prometheus'
+      orgId: 1
+      folder: ''
+      type: file
+      disableDeletion: false
+      editable: true
+      options:
+        path: /etc/grafana/provisioning/dashboards
+binaryData:
+  kuma-dataplane.json: {{ .DashboardDataplane }}
+  kuma-mesh.json: {{ .DashboardMesh }}
+  kuma-service-to-service.json: {{ .DashboardServiceToService }}
```

Comment:
Btw, the following Go template works for me: 
```yaml
apiVersion: v1
kind: ConfigMap
...
data:
  kuma-dataplane.json: |
{{ .DashboardDataplane | replace "${DS_PROMETHEUS}" "Prometheus" | indent 4 }}
  kuma-mesh.json: |
{{ .DashboardMesh | replace "${DS_PROMETHEUS}" "Prometheus" | indent 4 }}
  kuma-service-to-service.json: |
{{ .DashboardServiceToService | replace "${DS_PROMETHEUS}" "Prometheus" | indent 4 }}
```

So, it's not mandatory to use binary data in this situation.

And users, most likely, will appreciate better dashboards in plain text. This way they can run `kumactl install metrics` once, add the output to the source control, and keep tweaking dashboards in-place.

Reply:
Awesome!

Changed it, it works!